presto-main/etc/catalog/pinot.properties

```diff
@@ -2,3 +2,4 @@ connector.name=pinot
 
 # Pinot controller endpoint
 pinot.controller-urls=localhost:9000
+pinot.use-streaming-for-segment-queries=true
```

Comment:
Should we add a comment here that this would only work with versions of Pinot more than X ?

Reply:
make sense, I will comment out this line then.